test/from-draft3/to-draft4/type-has-objects-and-strings-in-its-array.json

```diff
@@ -0,0 +1,15 @@
+[
+    {
+        "title": "`type` is present and contains objects and strings in it",
+        "from": {
+            "type": [ {}, "string", "object", { "type": "string" } ],
+            "anyOf": [ { "type": [] } ]
+        },
+        "to": {
+            "type": [ {}, { "type": "string" } ],
```

Comment:
Draft 4 doesn't allow schemas in `type`

Reply:
Exactly. The point is that these tests are end-to-end tests, exercising ALL rules in conjunction vs individual rules. You want to test the system as a whole!